internal/pipe/aur/tmpl.go

```diff
@@ -0,0 +1,93 @@
+package aur
+
+type templateData struct {
+	Name            string
+	Desc            string
+	Homepage        string
+	Version         string
```

Comment:
Version needs to be stripped of any prefix letters. Things like `v1.0.0` trips up the version checker in `pacman`.

Reply:
that's ok, when it gets to this point, goreleaser already stripped out the `v` prefix :)